src/NUnitFramework/framework/Constraints/SubstringConstraint.cs

```diff
@@ -43,22 +44,34 @@ public SubstringConstraint(string expected) : base(expected)
         }
 
         /// <summary>
+        /// Modify the constraint to ignore case in matching.
+        /// </summary>
+        public override StringConstraint IgnoreCase
+        {
+            get { caseInsensitive = true; comparisonType = null; return this; }
```

Comment:
@nunit/framework-team Chime in please? Should we stay with what EqualConstraint does, or allow this behavior change and probably also change EqualConstraint?